ironfish-cli/src/utils/table.ts

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import { TimeUtils } from '@ironfish/sdk'
+import { table } from '@oclif/core/lib/cli-ux/styled/table'
+
+/**
+ * Estimated max length of the longest TimeUtils.renderTime()
+ */
+const MAX_TIMESTAMP_LENGTH = TimeUtils.renderString(
+  new Date(2024, 11, 25, 23, 59, 59).getTime(),
```

Comment:
Any special significance for this date and time? 😁

Reply:
I can write a note for that

Reply:
I wrote a note Explaining it.